tests/notifications.client.test.ts

```
@@ -511,5 +511,81 @@ describe("Notifications", () => {
         topics: [conversationTopic],
       });
     }, 10000); // Increase test timeout to 10 seconds
+
+    it.only("stops receiving notifications after unsubscribing", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)